documentation/src/main/asciidoc/public/reference/_compatibility.adoc

```diff
@@ -89,14 +89,25 @@ is also relevant for the OpenSearch distribution.
 [[compatibility-framework-quarkus]]
 === [[gettingstarted-framework-quarkus]] [[_quarkus]] Quarkus
 
-link:{quarkusUrl}[Quarkus] has an official extension for Hibernate Search with Hibernate ORM and Elasticsearch,
+link:{quarkusUrl}[Quarkus] has an official extension for <<mapper-orm,Hibernate Search with Hibernate ORM>>
+using the <<backend-elasticsearch,Elasticsearch backend>>,
 which is a tight integration with additional features, different dependencies, and different configuration properties.
 
 As your first step to using Hibernate Search within Quarkus,
 we recommend you follow Quarkus's https://quarkus.io/guides/hibernate-search-orm-elasticsearch[Hibernate Search Guide]:
 it is a great hands-on introduction to Hibernate Search,
 _and_ it covers the specifics of Quarkus.
 
+[[compatibility-framework-wildfly]]
+=== WildFly
+
+link:{wildflyUrl}[WildFly] includes modules for <<mapper-orm,Hibernate Search with Hibernate ORM>>
+using either the <<backend-lucene,Lucene backend>> or the <<backend-elasticsearch,Elasticsearch backend>>.
+
+To start using Hibernate Search within WildFly,
+see the https://docs.wildfly.org/30/Developer_Guide.html#using-hibernate-search[Hibernate Search section in the WildFly Developer Guide]:
```

Comment:
hmm I didn't see any "latest" or something like that link to the docs so that'd mean that we should keep track of it? 
I wonder if we maybe can add https://repo1.maven.org/maven2/org/wildfly/bom/wildfly/30.0.0.Final/wildfly-30.0.0.Final.pom to dependency management and parse the version out of it and use as a property?

Reply:
> hmm I didn't see any "latest" or something like that link to the docs so that'd mean that we should keep track of it?

Neither did I. They just don't have a "latest" URL.

>  I wonder if we maybe can add https://repo1.maven.org/maven2/org/wildfly/bom/wildfly/30.0.0.Final/wildfly-30.0.0.Final.pom to dependency management and parse the version out of it and use as a property?

I guess that would be considered a build dependency, then? Like Spring. Ok, I'll add that.

Reply:
Done! And I checked on my fork that Dependabot actually updates the dependency.